Services/ExamGenerator.Services.Data/ISubjectService.cs

```diff
@@ -9,5 +10,12 @@ namespace ExamGenerator.Services.Data
     public interface ISubjectService
     {
         public Task Add(SubjectInputModel model);
+        TDestination GetById<TDestination>(int subjectId);
+
+        Task Update(SubjectEditModel model);
+
+        Task<SubjectViewModel> Delete(int id);
+
+        Task<List<SubjectViewModel>> GetAllSubjects();
```

Comment:
Just GetAll. We are in subjectController, it is clear the GetAll will return subjects